babycenter/static/css/modules/_navigation.scss

```diff
@@ -1,5 +1,6 @@
 .navigation {
   a {
+    font-size: 14px;
```

Comment:
I don't think making the font-size smaller here is a good solution, as I want them to be nice and tappable.

Here are my suggestions:

```
.navigation {
   text-align: center;
  a {
    color: #666;
    font-weight: bold;
    white-space: nowrap;
  }
}
```